src/observables/ForkJoinObservable.ts

```diff
@@ -9,7 +9,7 @@ export class ForkJoinObservable<T> extends Observable<T> {
                                     Promise<any> |
                                     ((...values: Array<any>) => any)>) {
      super();
-   }
+  }
```

Comment:
What if I'd say now line 11 has wrong indentation, since `super()` is 3 spaces to the right? (1 too much)